{"name": "test_xfail", "status": "skipped", "statusDetails": {"message": "XFAIL reason: xfailing this test\n\n_pytest.outcomes.XFailed: xfailing this test", "trace": "def test_xfail():\n>       pytest.xfail(\"xfailing this test\")\nE       _pytest.outcomes.XFailed: xfailing this test\n\ntest.py:27: XFailed"}, "start": 1702571584324, "stop": 1702571584324, "uuid": "9ce9158f-1568-4952-8713-521c0e7dc714", "historyId": "9a4d0791413252253f88c96fb1cd5db7", "testCaseId": "9a4d0791413252253f88c96fb1cd5db7", "fullName": "test#test_xfail", "labels": [{"name": "suite", "value": "test"}, {"name": "host", "value": "byemen"}, {"name": "thread", "value": "33587-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test"}]}